script.js

```diff
@@ -0,0 +1,66 @@
+var xTurn = true;
+var oTurn = false;
+var oMoves = [];
+var xMoves = [];
+
+
+var winConditons = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+
+$('#reset').click(function () {
+  $('.box').removeClass('X, O').html('').addClass('clear')
+  xMoves = []
+  oMoves = []
+  xTurn = true;
+  oTurn = false;
+  whatPlayer();
+
+})
+//x o grid inputs
+$('.box').click(function(){
+  if (xTurn === true && ($(this).attr('class') === 'box'|| $(this).attr('class') ==='box clear' )) {
+  //add X class and x to box
+  $(this).addClass('X').html('X')
+  xMoves.push(parseInt($(this).attr('data-num')))
+  console.log(xMoves, 'x');
+  oTurn = true;
+  xTurn = false;
+  whatPlayer();
+  winnerCheck()
+
+
+} else if (oTurn === true && ($(this).attr('class') === 'box' || $(this).attr('class') ==='box clear' )) {
+    //add O class and o to box
+    $(this).addClass('O').html('O')
+    oMoves.push(parseInt($(this).attr('data-num')))
+    console.log(oMoves, 'o');
+    oTurn = false;
+    xTurn = true;
+    whatPlayer();
+    winnerCheck();
+  }
+});
+function winnerCheck() {
+  
+  if (winConditons.indexOf(xMoves) > -1){
+    alert('winner x')
```

Comment:
Is there a better way you could get this information to the player than an alert?